package.json

```diff
@@ -36,5 +36,9 @@
 		"safe-publish-latest": "^1.1.4",
 		"tape": "^4.13.0"
 	},
-	"license": "MIT"
+	"license": "MIT",
+	"files": [
+		"index.js",
+		"component.json"
```

Comment:
this file wouldn't need to be published anyways, it's for a long-obsolete package manager that installs from github.

Reply:
Thanks for the clarification! I removed it.